conf/GettingStartedDB.properties

```diff
@@ -0,0 +1,3 @@
+base.dir=../../pentaho-solutions/system/GettingStartedDB
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.